deploy/disk_type_labels.sh

```diff
@@ -0,0 +1,384 @@
+#!/bin/bash
+
+# Ensure we are running Bash 4.0 or higher for associative arrays
+if ((BASH_VERSINFO[0] < 4)); then
+    echo "Error: This script requires Bash 4.0 or higher." >&2
+    exit 1
+fi
+
+# ==========================================
+# 0. Argument Parsing & Usage
+# ==========================================
+
+usage() {
+    echo "Usage: $0 [OPTIONS]"
+    echo ""
```

Comment:
Can you add a line explaining how this is used? something along the lines of, if my-k8s-node has machine type X, then kubectl label node my-k8s-node $($0` -m X) will add the correct label.